src/sentry/static/sentry/app/views/organizationIntegrations/integrationPluginRow.tsx

```diff
@@ -0,0 +1,111 @@
+import {withTheme} from 'emotion-theming';
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from '@emotion/styled';
+import Link from 'app/components/links/link';
+import {PanelItem} from 'app/components/panels';
+import {t} from 'app/locale';
+import CircleIndicator from 'app/components/circleIndicator';
+import PluginIcon from 'app/plugins/components/pluginIcon';
+import SentryTypes from 'app/sentryTypes';
+import space from 'app/styles/space';
+import {PluginWithProjectList} from 'app/types';
+
+type Props = {
+  plugin: PluginWithProjectList;
+  legacy: boolean;
+};
+
+export default class PluginRow extends React.Component<Props> {
+  static propTypes = {
+    plugin: PropTypes.object.isRequired,
+  };
+
+  static contextTypes = {
+    organization: SentryTypes.Organization,
+  };
+
+  get isEnabled() {
+    return this.props.plugin.projectList.length > 0;
```

Comment:
@NisanthanNanthakumar Strictly speaking, it's possible to only have items in `projectList` that are disabled configs (`enabled=false`). But for the purpose of showing things that are installed, that might be OK. Probably worth a comment explaining this.